org.alienarena.alienarena.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.alienarena.alienarena</id>
+  <launchable type="desktop-id">alien-arena.desktop</launchable>
+  <name>Alien Arena</name>
+  <summary>Ultra fast paced arena FPS</summary>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">mild</content_attribute>
+    <content_attribute id="language-profanity">mild</content_attribute>
+    <content_attribute id="language-humor">mild</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
+    <content_attribute id="money-advertising">mild</content_attribute>
+  </content_rating>
+  <description>
+    <p>Do you like old school deathmatch with modern features? How about rich, colorful, arcade-like atmospheres? How about...retro Sci Fi? Then you're going to love what Alien Arena has in store for you! This game combines some of the very best aspects of such games as Quake III and Unreal Tournament and wraps them up with a retro alien theme, while adding tons of original ideas to make the game quite unique.</p>
+    <p>Official community website: <a href="https://www.alienarena.org/">https://alienarena.org</a></p>
```

Comment:
I am pretty sure the validation will fail because of the links.

Reply:
Is there another way i should do the links? I put it through flatpak run org.freedesktop.appstream-glib validate and it didn't say anything about that.

Reply:
it it passes then fine.

Reply:
It's what `<url type="homepage">` is for. I'd also remove URLs to YouTube, I haven't seen any app do that.